<launch>
  <node pkg="iahrs_ros" type="iahrs_ros" name="iahrs" output="screen">
    <param name="port" value="/dev/ttyUSB0" />
    <param name="baud_rate" value="115200" />
    <param name="comm_recv_timeout" value="30" />
    <param name="loop_rate" value="40" />
    <param name="sync_mode" value="true" />
  </node>
</launch>
